feat: add Kotlin for Data Analysis section (#4125)
JetBrains/kotlin-web-site: docs/topics/data-analysis/kotlin-notebook-add-dependencies.md

[//]: # (title: Add dependencies to your Kotlin Notebook)

<microformat>
   <p>This is the third part of the <strong>Getting started with Kotlin Notebook</strong> tutorial. Before proceeding, make sure you've completed the previous steps.</p>
   <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="kotlin-notebook-set-up-env.md">Set up an environment</a><br/>
      <img src="icon-2-done.svg" width="20" alt="Second step"/> <a href="kotlin-notebook-create.md">Create a Kotlin Notebook</a><br/>
      <img src="icon-3.svg" width="20" alt="Third step"/> <strong>Add dependencies to a Kotlin Notebook</strong><br/>      
      <img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Share a Kotlin Notebook<br/>
  </p>
</microformat>

You've already created your first Kotlin Notebook! Now let's learn how to add dependencies to libraries, which
is necessary to unlock advanced features.

> > The Kotlin Standard Library can be used out of the box, so you don't have to import it.

Reviewer: I've seen both in the docs 😕 - which one is correct? in the API reference ( & in the overview of docs) and in Maven central it is written as Kotlin Standard Library - but I've seen Kotlin standard library as well.

```kotlin
// Replace libraryName with the library dependency you want to add
%use libraryName
```

You can also use the autocompletion feature in Kotlin Notebook to quickly access available libraries.
![Autocompletion feature in Kotlin Notebook](autocompletion-feature-notebook.png){width=700}

## Add Kotlin DataFrame and Kandy libraries to your Kotlin Notebook

Let's add two popular Kotlin library dependencies to your Kotlin Notebook:
* The [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html) gives you the power to manipulate data in your Kotlin projects. 
You can use it to retrieve data from [APIs](data-analysis-work-with-api.md), [SQL databases](data-analysis-connect-to-db.md), and [various file formats](data-analysis-work-with-data-sources.md), such as CSV or JSON.
* The [Kandy library](https://kotlin.github.io/kandy/welcome.html) provides a powerful and flexible DSL for [creating charts](data-analysis-visualization.md).

To add these libraries:

1. Click **Add Code Cell** to create a new code cell.
2. Enter the following code in the code cell:

```kotlin

> // Ensures that we use the latest available versions of libraries

Reviewer: ```suggestion
// Ensures that the latest available libraries versions are used
```
